src/AppBundle/Controller/AssetsController.php

```diff
@@ -38,4 +40,23 @@ public function assetAction($path, Request $request)
 
         return $response;
     }
+
+    /**
+     * @Route("/maps/{latitude},{longitude}", requirements={
```

Comment:
Trying to be consistent, so do you confirm the change?

https://github.com/EnMarche/en-marche.fr/pull/173/files/ec0a78023b2af7aab27e1c8b03c254edaff56c22#diff-5a328f7080fed0772772dfbf0fe019a4R19